charactersheet/charactersheet/utilities/convineince.js

```diff
@@ -4,11 +4,13 @@
  * This file contains a number of generic utility functions used throughout
  * Adventurer's Codex
  *
- * author: Brian Schrader
+ * @author Brian Schrader
+ * @author Nathaniel Arellano
  */
 
 var Utility = {
-    markdown: {}
+    markdown: {},
+    stringUtil: {}
```

Comment:
I'd prefer if this was just `string`. Reads better to me. It's already in `Utilities`, no need to say `util` in the name.

`Utilities.string.truncateStringAtLength(string, boundary)`